src/main/java/frc/team2412/robot/commands/intake/IntakeDefaultCommand.java

```diff
@@ -1,17 +1,13 @@
 package frc.team2412.robot.commands.intake;
 
-import static frc.team2412.robot.subsystems.IntakeSubsystem.IntakeConstants.INTAKE_HOLD_SPEED;
-
 import frc.team2412.robot.subsystems.IntakeSubsystem;
-import frc.team2412.robot.subsystems.IntakeSubsystem.IntakeConstants.GamePieceType;
 
 public class IntakeDefaultCommand extends IntakeSetCommand {
 
 	public IntakeDefaultCommand(IntakeSubsystem intakeSubsystem) {
 
-		super(
-				intakeSubsystem,
-				intakeSubsystem.detectType() == GamePieceType.NONE ? 0 : INTAKE_HOLD_SPEED);
+		super(intakeSubsystem, 0);
+		setSpeed(intakeSubsystem.getHoldSpeed());
```

Comment:
What was the reason for not setting this in the call to the super constructor? Should this be continually set (in execute)?

Reply:
Setting the method in the constructor doesn't seem to actually work correctly (tried running default command with game piece in and doesnt work), i definitely should use execute though.